shopfloor/models/shopfloor_log.py

```diff
@@ -1,130 +1,41 @@
 # Copyright 2020 Camptocamp SA (http://www.camptocamp.com)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
-import logging
-from datetime import datetime, timedelta
-
-from odoo import api, fields, models, tools
-
-_logger = logging.getLogger(__name__)
+from odoo import models
+
+# TODO: drop this model on next release.
+# The feature has been moved to `rest_log`.
+#
+# It has been tried several times
```

Comment:
@simahawk Do you've tried to create a migration script using the methods [`rename_models`](https://github.com/OCA/openupgradelib/blob/master/openupgradelib/openupgrade.py#L747) and [`rename_tables`](https://github.com/OCA/openupgradelib/blob/master/openupgradelib/openupgrade.py#L683) from openupgradelib?

Reply:
yes, but the issue is not there. The issue is that when the ORM tries to delete the field options stored in ir.model.fields.selection it breaks because it does not find the model :man_shrugging: 
I didn't want to spend time on digging into the issue tho.